src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor();
+        Random random = new Random();
```

Comment:
You should not create a new instance of 'Random' every time the method is called. It's better to have it as a private class-level field and initialize it only once.